### ISSUE TITLE
Can only stop sprinting by stopping walking at all

### ISSUE THREAD

**Issue by TheEngineer404** [FinalForEach/Cosmic-Reach-Issue-Tracker#103]

**Comment by ArnulfoRuiz** [comment]
This also happens in Minecraft, which is the inspiration for the game, so it's intentional.
If you think it is too, close the issue.

**Comment by Fesiug** [comment]
Personally I like this feature as I don't have to continually hold down the sprint key

**Comment by mellamokb** [comment]
That is true but what is different from minecraft is that you stop sprinting once you hit a block, that doesn't happen in Cosmic Reach.
